client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import FooterSection from 'calypso/components/footer-section';
+import LinkCard from 'calypso/components/link-card'; // should we host this one inside footer-section?
+
+interface Article {
+	title: string;
+	category: string;
+	url: string;
+	background: string;
+}
+
+const deslugify = ( string: string ) => string.replaceAll( '-', ' ' );
+const slugify = ( string: string ) =>
+	string
+		.replace( /[&/\\#,+()$~%.'":*?<>{}’]/g, '' )
+		.toLowerCase()
+		.replaceAll( ' ', '-' );
+
+const ArticleLinksContainer = styled.div`
+	@media ( max-width: 960px ) {
+		grid-template-columns: repeat( 1, 1fr );
+	}
+
+	@media ( max-width: 660px ) {
+		padding: 0 16px;
+	}
+
+	display: grid;
+	grid-template-columns: repeat( 3, 1fr );
+	column-gap: 29px;
+	row-gap: 10px;
+`;
+
+const EducationFooter = () => {
+	const translate = useTranslate();
+
+	const articles: Article[] = [
+		{
+			category: translate( 'website-building', {
+				comment:
+					'Wordpress.com Go article (https://wordpress.com/go/) category on kebab case, example: Website Building = website-bulding',
+				textOnly: true,
+			} ),
```

Comment:
Can I suggest we just put the text we want in here?

```suggestion
			category: __( 'Website Building' ),
```

I'm thinking you might want to load these in from the go site's rest api at some point, for now lets just assume it can send back translated taxonomy names if you send it a locale with the request.

Reply:
Makes sense

Reply:
✅